common/src/credentials/mod.rs

```diff
@@ -71,7 +71,7 @@ impl Labeled for AsCredentialBody {
 }
 
 fn legacy_credential_hash(payload: &impl tls_codec::Serialize, label: &str) -> [u8; HASH_SIZE] {
-    let hash_label = format!("Infra Credential Fingerprint {label}");
+    let hash_label = format!("Air Credential Fingerprint {label}");
```

Comment:
Do you think we can just remove the "Air" prefix from here? I am not sure there is any benefit of having it here.